### ISSUE TITLE
Error 404 from the plugin

### ISSUE THREAD

**Issue by ghost** [stcr/subscribe-to-comments-reloaded#387]
Hello, 
Sorry for my english I'm french.
This morning I checked Google Search Console and I realised that I have some news 404 errors. All of them are comment-subscriptions. I think my plugin isn't right configurated. 
How can I fix this ? 
Thanks for your help, 
Bests Regards !
[comment subscriptions](https://user-images.githubusercontent.com/35599780/35150566-28b2988a-fd1b-11e7-86a5-f78527169e12.PNG)

Mathilde

**Comment by Reedyseth** [comment]
Hello @Mathilde295 , The error 404 is very common on the Google Search Console. It records every URL on the web. But it says de 404 error because the URL of the manage subscription page changes once you change the Plugin key.

![image](https://user-images.githubusercontent.com/1228711/35171247-586363c0-fd20-11e7-9740-bb218b24cf89.png)

What I suggest is to add the option to ignore this error on the Google Search Console

**Comment by Mathilde295** [comment]
Thanks for your reply. But I haven't change the key and it was working for many months. 
People can't access on the virtual page because it's an error 404. You can check it on my website if you want more informations : https://maviedecoeliaque.fr 
Peope can't access on any of the links of the plugin because they redirect to an error 404.

**Comment by Reedyseth** [comment]
Okay, let me check it out !

**Comment by Mathilde295** [comment]
Thank you so much ! :)

**Comment by Reedyseth** [comment]
Do you have the Virtual Page Enabled?

![image](https://user-images.githubusercontent.com/1228711/35173567-19a8fde4-fd29-11e7-93c4-b31ed74eb38e.png)

**Comment by Mathilde295** [comment]
Yes it is 
[capture](https://user-images.githubusercontent.com/35599780/35173619-56bdab1a-fd6c-11e7-956e-4bdcbd08c226.PNG)

**Comment by Reedyseth** [comment]
Can you send me the information on the text box under the StCR System tab to my email?

![image](https://user-images.githubusercontent.com/1228711/35173680-7e796bfa-fd29-11e7-8908-c00fdd0a5813.png)

reedyseth at] gmail [dot) com

**Comment by Mathilde295** [comment]
Yes I will do it, thanks :)

**Comment by tahmasebi** [comment]
Hi @Reedyseth 
I have the same problem after new update. could you tell us what was the problem and how to we should solve it?

**Comment by Reedyseth** [comment]
Hi @tahmasebi , Something that I found is that having an InnoDB machine on the StCR table is also causing this. In order to indentify this you might check the error logs of your hosting / server.

Please help me filling up he following information on this page:

http://subscribe-reloaded.com/support-request-system-info/

**Comment by Nick776** [comment]
Hey @Reedyseth we got a strange effect. We are uninstall StRC two year ago but now we got a lot of 404 in the new Google Search Console like this:

https://www.hello-world.net/comment-subscriptions/?srp=1740&srk=e906dff0b3d7b39082df98586ca98dd7&sra=s&srsrc=f

Can you help us to get rid of this problem?

thnx in advance & best regards from Germany
Nico

**Comment by Reedyseth** [comment]
@Nick776 You should use Google Webmaster Tools to block access to `?srp=`:
https://support.google.com/webmasters/topic/4598466?rd=1

For example, you can add the following to your `robots.txt` file to block the Google Web Crawler from indexing specific pages:

```
User-agent: *
Disallow: /comment-subscriptions*?srp=*

**Comment by Nick776** [comment]
hey @Reedyseth . Thx for you fast reply. Unfortunately Google changed to pay attention for "noindex" directives in the robots.txt as you can read it here... :-((

https://searchengineland.com/google-to-stop-supporting-noindex-directive-in-robots-txt-319003

**Comment by Reedyseth** [comment]
@Nick776 Yes de `X-Robots-Tag: noindex` option was disabled but using the `Disallow` directive will help you although they state:
> **Disallow in robots.txt**: Search engines can only index pages that they know about, so blocking the page from being crawled usually means its content won’t be indexed.  While the search engine may also index a URL based on links from other pages, without seeing the content itself, we aim to make such pages less visible in the future.

So in that case your best option will be to install a redirection plugin and add the rule `/comment-subscriptions*?srp=*` to point to a 404 redirection. This will tell the crawlers to avoid indexing this.
